### ISSUE TITLE
skipWords & capital letters in parameter names

### ISSUE THREAD

**Issue by kachkaev** [aotaduy/eslint-plugin-spellcheck#5]
Just got into this situation:

```json
"skipWords": [
    "blahblah"
]
```

```javascript
a.isBlahblah
// You have misspelled word: Blahblah on Identifier at line X col Y
```

Capitalising ```blahblah``` in the plugin config solves the issue, but raises new warnings if the word is somewhere in comments. Having both ```blahblah``` and ```Blahblah``` in the config seems too explicit.

It would be nice if checking of camel-case identifiers was done in a slightly smarter way.

Cheers!

**Comment by aotaduy** [comment]
You are right I have updated the rules checking only for lowercase will be avilable in next npm version to be release tonight or over the weekend. 
In other library  [(lintspelljs)](https://github.com/aotaduy/lintspelljs/blob/master/lintspell-lib.js) I have a better method to check for this but did'nt want to added the dependency on this eslint rule

**Comment by aotaduy** [comment]
Updated npm to 0.0.6

**Comment by kachkaev** [comment]
Great! There's no need to declare a term in ```skipwords``` more than once – the checker successfully matches words written in any case correctly.

As a further improvement, it would be nice if capitalized words in ```skipwords``` expected all occurrences not to be lower-case. E.g.;

 ```js
"skipWords": [
    "Andres"
]
```

 ```js
const a = 'Andres'; // no warning as expected
const b = 'ANDRES'; // no warning as expected
const c = 'andres'; // I'd expect an error here, but now there isn't any
```

Generally speaking, any letter in a word that's upper-case in ```skipwords``` always has to be upper-case. Just like it is now for the standard dictionary:
 ```js
const a = 'NASA'; // no warning as expected
const b = 'Nasa'; // warning as expected
const c = 'nasa'; // warning as expected
```

I'm not sure how easy it would be to implement this though.
